brev-bestilling-boot/src/main/resources/application-local.yml

```diff
@@ -0,0 +1,3 @@
+brevmetadata:
```

Comment:
Den er generelt greit, men vi kan ikke mocke alt så trenger vi noe som gårogså mot ete tjenester. Vi har opprettet en VDI profil. Har du tested hvis BrevMetadata er tilhjengelig fra VDI ?

Reply:
Når jeg starter med -Dspring.profiles.active=vdi da går den mot `https://pensjon-brevmetadata.dev.adeo.no/api/brevdata` samme hvis jeg starter uten aktiv profil, da går samme url `https://pensjon-brevmetadata.dev.adeo.no/api/brevdata`
Hvis starter med -Dspring.profiles.active=local da går mot `http://localhost:6060/api/brevdata` (som ligger i application-local.yml)
application-local.yml er lastet automatisk hvis set aktiv profil til "local" og trenger ikke mocke hver verdi, bare de som vi har lyst å ha spesielt for local profil.

Reply:
Det var bare spørsmål hvis GCP er tilgjengelig fra VDI. Jeg har ikke tested det.

Reply:
vi har flere instanser med samme ingress, så mulig at den i vdi går mot dev-fss (aka on premise).